pkg/processor/trigger/cron/trigger.go

```diff
@@ -189,6 +185,17 @@ func (c *cron) getMissedTicks(schedule cronlib.Schedule, nextEventSubmitTime tim
 	return missedTicks
```

Comment:
when returning here, missedTicks must always be 0 right, I think it'll be better to change this to `return 0`